### ISSUE TITLE
Animated avatars

### ISSUE THREAD

**Issue by snan** [metabolist/metatext#52]
I have "Animate avatars" set to "never" but this avatar animation is still playing: https://cybre.space/@nleigh (in Metatext; it is static in Safari).

**Comment by mdoshi1** [comment]
Hello, @snan! Is this issue still happening for you? I just checked this profile on version 1.6.0. The avatar does not animate when I have "Animate avatars" set to "never". Otherwise, it does animate.

<img src="https://user-images.githubusercontent.com/9256511/202652731-552fcd5d-5dbd-4fc6-9959-10fe401f29a4.PNG" width="250">

<video src="https://user-images.githubusercontent.com/9256511/202652771-85c42447-0047-4ad9-bc94-de321c0b3e4a.mov">
</video>

**Comment by snan** [comment]
Yes, it still animates for me on 1.6.0. with animations set to never.
